plugins/arElasticSearchPlugin/config/mapping.yml

```diff
@@ -459,8 +459,19 @@ mapping:
         properties:
           id: { type: integer }
           slug: { type: keyword }
+      repository:
+        _i18nFields: [authorizedFormOfName]
+        dynamic: strict
+        properties:
+          id: { type: integer }
+          slug: { type: keyword }
+          identifier: { type: keyword }
+      names:
+        _i18nFields: [authorizedFormOfName]
+        dynamic: strict
+        properties:
+          id: { type: integer }
     _foreign_types:
-      repository: repository
       names: actor
```

Comment:
Should we remove the names from the full foreign types? This is probably adding the old names mapping and overwriting the new one from the partial foreign types.

Reply:
Good catch @jraddaoui, thanks.  I've removed the `names: actor` line from the `_foreign_types` definitions.

Reply:
It's not ideal but I think it's okay to have the old mapping and only add the partial data if it's not easy to reflect the changes in the names mapping, in which case you could remove the partial foreign type from the mapping instead of the other way around.

Reply:
@jraddaoui I think you are right in removing the `names: actor` from the full foreign types definitions.  I've checked the resulting data in ES (via the JSON-LD REST endpoint) and the expected "names" data is present without effecting other actor types (e.g. creator, inherited_creator).